src/wallet/wallet.h

```diff
@@ -1147,6 +1147,16 @@ class CWallet final : public FillableSigningProvider, private interfaces::Chain:
      */
     bool CreateTransaction(interfaces::Chain::Lock& locked_chain, const std::vector<CRecipient>& vecSend, CTransactionRef& tx, CAmount& nFeeRet, int& nChangePosInOut,
                            std::string& strFailReason, const CCoinControl& coin_control, bool sign = true);
+    /**
+     * Submit the transaction to the node's mempool and then relay to peers.
+     * Should be called after CreateTransaction unless you want to abort
+     * broadcasting the transaction.
+     *
+     * @param tx[in,out] The transaction to be broadcast.
```

Comment:
in commit c7397e074996086578d9ac3db78d654fdbbf58a5:

this is not a return parameter. The tx is consumed (`std::move`)

Reply:
Thanks. Fixed.